app/controllers/user_sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UserSessionsController < ApplicationController
+  skip_before_action :require_login, except: [:destroy]
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    if @user = login(params[:email], params[:password])
+      flash[:success] = "Добро пожаловать!"
+      redirect_to root_path
+    else
+      flash[:danger] = "Попробуй еще раз или зарегистируйся!"
+      render 'new'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.